cheetah/particles/particle_beam.py

```diff
@@ -1074,6 +1074,13 @@ def momenta(self) -> torch.Tensor:
         """Momenta of the individual particles."""
         return torch.sqrt(self.energies**2 - electron_mass_eV**2)
 
+    def clone(self) -> "ParticleBeam":
+        return ParticleBeam(
+            particles=self.particles.clone(),
+            energy=self.energy.clone(),
+            particle_charges=self.particle_charges.clone(),
+        )
```

Comment:
I suppose the survival probability is missing from this list? We merged the other PR here after I had written the `clone` method.